lib/auth/auth_login_form.dart

```diff
@@ -0,0 +1,100 @@
+
+import 'package:easybreezy_app/auth/bloc/auth_bloc.dart';
+import 'package:easybreezy_app/auth/modals/modals.dart';
+
+class AuthLoginForm extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => _AuthLoginFormState();
+}
+
+class _AuthLoginFormState extends State<AuthLoginForm> {
+  final GlobalKey<FormBuilderState> _fbKey = GlobalKey<FormBuilderState>();
+  bool _autoValidate = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocConsumer<AuthBloc, AuthState>(
+      listener: (context, state) {
+        if (state is AuthSignInSuccess) {
+          Navigator.of(context).pushNamed('/');
+        }
+      },
+      builder: (context, state) {
+        final bool isAuthSignInFailure = state is AuthSignInFailure;
+        return Padding(
+            padding: EdgeInsets.all(16),
+            child: FormBuilder(
+                autovalidate: _autoValidate,
+                key: _fbKey,
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: <Widget>[
+                    FormBuilderTextField(
+                      attribute: "email",
+                      decoration: InputDecoration(
+                        icon: Icon(Icons.email),
+                        labelText: "Email",
+                        errorText: isAuthSignInFailure ? "" : null,
+                      ),
+                      validators: [
+//                        FormBuilderValidators.required(errorText: "Required"),
+//                        FormBuilderValidators.email(errorText: "Invalid")
+                      ],
+                    ),
+                    FormBuilderTextField(
+                      attribute: "password",
+                      decoration: InputDecoration(
+                        icon: Icon(Icons.lock),
+                        labelText: "Password",
+                        errorText: isAuthSignInFailure ? "" : null,
+                      ),
+                      validators: [
+                        FormBuilderValidators.required(errorText: "Required"),
+                      ],
+                    ),
+                    Container(
+                      margin: EdgeInsets.only(top: 30),
+                      child: state is AuthSignInFailure
+                          ? _getErrorWidget(state.error)
+                          : null,
+                    ),
+                    Padding(
+                      padding: EdgeInsets.only(top: 16),
+                      child: RaisedButton(
+                        color: Theme.of(context).buttonColor,
+                        onPressed: () => _validateInputs(),
+                        child: Text("Log in"),
+                      ),
+                    ),
+                  ],
+                )));
+      },
+    );
+  }
+
+  Widget _getErrorWidget(String text) {
```

Comment:
Let's try to extract `ErrorMessage` to common components

Reply:
Moved ErrorMessage into shared components